platform/core/src/extensions/ExtensionManager.ts

```diff
@@ -331,7 +340,7 @@ export default class ExtensionManager {
     }
 
     try {
-      const extensionModule = getModuleFn({
+      const extensionModule = extension[getModuleFnName]({
```

Comment:
Why was this changed?

Reply:
Because it doesn't associate the "this" property with the extension it is running, so if you need to store some variables
between init and other functions, then without this change, you can't do it.

Reply:
Cool. Thanks. Consider adding a comment maybe?